crates/ruff_linter/src/rules/pycodestyle/rules/literal_comparisons.rs

```diff
@@ -103,27 +109,31 @@ impl AlwaysFixableViolation for TrueFalseComparison {
         let TrueFalseComparison(value, op) = self;
         match (value, op) {
             (true, EqCmpOp::Eq) => {
-                format!("Comparison to `True` should be `cond is True` or `if cond:`")
+                format!("Instead of directly comparing to `True`, use `if cond:` for truth checks")
```

Comment:
Hmm, I have a slight preference for the existing messages with these. `if cond is True` is something you (occasionally) need to write, and that's still a "direct comparison".

Maybe something like this? (And ditto for the other branches)

```suggestion
                format!("Avoid equality comparisons to `True`; use `if cond:` for truth checks")
```

Reply:
Added suggestion.